### ISSUE TITLE
Lightsource which illuminates larger area than default

### ISSUE THREAD

**Issue by Xalcon** [Xalcon/TorchMaster#5]
From "ReiDaTecnologia" (Curseforge comment):
> I have a Request/Idea:
> You can put the option of it consume energy (RF, Tesla etc) to illuminate an area larger than by default and can set how much can light up and the larger the area of lighting the more energy you consume?
> Thanks!

I will probably implement this in multiple steps (if possible). First step will be just the light source, working similar to the [wrath lamp](http://feed-the-beast.wikia.com/wiki/Wrathlamp) from factorization. In a later release I might add support for energy systems.

**Comment by Xalcon** [comment]
![image](https://cloud.githubusercontent.com/assets/5042450/23340029/183f0b2a-fc2f-11e6-85e3-f2112067692d.png)
I've implemented the first iteration of the dread lamp. It will illuminate a 15x15x15 area with a light level auf 15 (max) using invisible light sources. Currently the light sources are only placed but not removed. There is also no line of sight check and light sources are not replaced if they got removed by other means (flowing fluids, block placed and removed, etc).

The [Wrath Lamp](https://ftbwiki.org/Wrathlamp) from factorization is frequently checking the light volume for missing light blocks. We need to specify how often those checks should occur. Also, do we need line of sight checks? (LoS)
